src/test/rustdoc-gui/anchors.goml

```diff
@@ -0,0 +1,12 @@
+goto: file://|DOC_PATH|/test_docs/struct.HeavilyDocumentedStruct.html
+
```

Comment:
Better enforce the theme used (I had some issues because my default theme isn't the light one). To do it, take a look [here](https://github.com/rust-lang/rust/blob/master/src/test/rustdoc-gui/code-color.goml#L24-L27).